### ISSUE TITLE
For training-validation question

### ISSUE THREAD

**Issue by Jingling1** [NVlabs/EmerNeRF#13]
Thanks for your wonderful work!

Could you please take little time to answer my question about training-validation? Looking forward to hearing from you!

To the best of my knowledge, NeRF typically requires retraining for new data in order to encode specific characteristics of the data into network parameters. So there is a parameter in the code called scene_ IDX, which specifies specific data.

May I ask if my above understanding is correct?

However, what confuses me is that the code comments are "# which scene to use, [0, 798] for waymo's training set and [0, 849] for nuscenes's train/val sets, inclusive”. It indicates the validation set that can be used for nuscenes, does this mean that your model can generalize to new data?
